services/core-api/app/api/projects/project/project_util.py

```diff
@@ -6,8 +6,11 @@
 
 class ProjectUtil:
 
-    def notifiy_file_updates(project, mine):
-        if is_feature_enabled(Feature.MINE_APPLICATION_FILE_UDPATE_ALERTS):
+    @staticmethod
+    def notify_file_updates(project, mine, status_code = None):
```

Comment:
```suggestion
    def notify_file_updates(cls, mine, status_code = None):
```

Then i would recommend this.

Reply:
the class isn't actually a project though. It's a completely different class.
Like I'd actually have to change it to `def notify_file_updates(cls, project, mine, status_code):` and not use the cls parameter. 

I think I actually want to refactor the file to be more like `services/core-api/app/activity/utils.py`

Reply:
I'm totally wrong. you've got this right.